### PR TITLE
Bugfix/dev 297

### PR DESCRIPTION
**Pull request by zainzafar90**

### CODE REVIEW

File: src/app/mail/mail-detail/mail-detail.component.ts [hunk 1]
```diff
@@ -35,6 +35,8 @@ export class MailDetailComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
+    console.log(this.mailFolder)
```
Comment (Waseem826) [inline src/app/mail/mail-detail/mail-detail.component.ts]:
```suggestion

```

**Comment by Waseem826** [comment]
This breaks the app. Load the app as non-logged in user and try to login. There will be an exception in the console and you won't be logged in

**Comment by Waseem826** [comment]
Also, previously the "Help" button was hidden for logged in user but now its always shown. Is that the new requirement?

**Comment by atifsaddique211f** [comment]
> This breaks the app. Load the app as non-logged in user and try to login. There will be an exception in the console and you won't be logged in

@zainzafar90 can you please take care of this?

**Comment by atifsaddique211f** [comment]
> Also, previously the "Help" button was hidden for logged in user but now its always shown. Is that the new requirement?

It should be hidden for logged in users.

**Comment by Waseem826** [comment]
All previous issues are resolved but I've found another issue.
- Go to login page
- Click help button (wait for it to load)
- Login
- Logout
- Click on Help button
The help button disappears

**Comment by zainzafar90** [comment]
> All previous issues are resolved but I've found another issue.
> 
> * Go to login page
> * Click help button (wait for it to load)
> * Login
> * Logout
> * Click on Help button
> 
> The help button disappears

Waseem Can you test it again? Thanks

**Pull request merged**